# Superpy_advance-time
advance-time error
PS C:\Users\Administrator\Winc\superpy> python main.py advance-time 2
Traceback (most recent call last):
  File "C:\Users\Administrator\Winc\superpy\main.py", line 114, in <module>
    main()
  File "C:\Users\Administrator\Winc\superpy\main.py", line 59, in main     
    if args.action == 'products':
       ^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'action'
